### PR TITLE
feat: add dialog to add a AboutUs Modal

### PR DESCRIPTION
**Pull request by Sebastien-Ahkrin**
close: https://github.com/zakodium-oss/analysis-ui-components/issues/376

Capture d’écran 2022-11-21 à 15 57 43

### CODE REVIEW

File: src/app/about/AboutDialogToolbarButton.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  `,
+};
+
+export function AboutDialogToolbarButton(props: AboutDialogToolbarButtonProps) {
+  const { name, icon, body } = props;
+  const [isOpenDialog, openDialog, closeDialog] = useOnOff(false);
+
+  return (
+    <>
+      <Toolbar.Item title={name} onClick={openDialog}>
```
Comment (targos) [inline src/app/about/AboutDialogToolbarButton.tsx:26]: Should be "About ${name}"

File: src/app/about/AboutDialogToolbarButton.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        {icon}
+      </Toolbar.Item>
+      <Modal isOpen={isOpenDialog} onRequestClose={closeDialog}>
+        <Modal.Header>Analysis UI Components - BIG MAP</Modal.Header>
```
Comment (targos) [inline src/app/about/AboutDialogToolbarButton.tsx:30]: Should be "${name}"

File: src/pages/big-map/MainLayout.tsx [hunk 3]
```diff
@@ -98,3 +105,56 @@ export default function MainLayout() {
...
+  `,
+  separator: css`
+    border-bottom: 1px solid gray;
+    width: 15px;
+    height: 1px;
+    margin: 10px 0px;
+  `,
+};
+
+function AboutInformation() {
```
Comment (targos) [inline src/pages/big-map/MainLayout.tsx:131]: Please move this component to a new file

**Review by targos — CHANGES_REQUESTED** [review]
Please also add an example about dialog to the demo app
You can put it instead of the useless logo: 
image

File: src/pages/big-map/MainLayout.tsx (hunk 3 above)
Comment (targos) [reply src/pages/big-map/MainLayout.tsx]: I mean a new file in the big-map folder.
The about dialog will be specific to this application.

**Comment by Sebastien-Ahkrin** [comment]
> 

where did you took this screen ?

**Pull request merged**